.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
-sudo: false
 cache: pip
-
+ 
```

Comment:
There is still this white space that's not required. Please remove this white space change as well.
(Usually don't make whitespace changes!)

Reply:
Okay